fronts-client/src/bundles/collectionsBundle.ts

```diff
@@ -83,9 +83,9 @@ const collectionActions = {
 type CollectionActions = Actions<Collection> | SetHidden;
 
 const collectionReducer = (
-  state: State<Collection>,
+  state: LibState<Collection>,
```

Comment:
Not necessarily in this PR, but may be worth looking at why some of our slices `State` of state need this pattern.

Reply:
`asyncResourceBundles` could do with some documentation in here